modules/electrophysiology_browser/ajax/get_electrophysiology_session_data.php

```diff
@@ -0,0 +1,203 @@
+<?php
```

Comment:
```suggestion
<?php declare(strict_types=1);
```

Reply:
Thank you! Will be added in the next commit.